sensor_msgs_py/test/test_point_cloud2.py

```diff
@@ -0,0 +1,111 @@
+
+
```

Comment:
Please put the BSD license at the top here.

Reply:
Since I wrote this code, am I given the copyright? Or do I, by contributing, give the copyright to OSRF?

Reply:
> Since I wrote this code, am I given the copyright? Or do I, by contributing, give the copyright to OSRF?

You maintain copyright to it, since you wrote it.